lk2nd/device/dts/msm8916/msm8916-samsung.dts

```diff
@@ -490,6 +490,25 @@
 
 	/* rev 4 */
 
+	j5xnlteb {
```

Comment:
Why?

```suggestion
	j5xnltedx {
```

Reply:
oops, was writing the codename from the top of my head